qbft/spectest/tests/proposal/different_justifications.go

```diff
@@ -49,10 +48,10 @@ func DifferentJustifications() tests.SpecTest {
 	return &tests.MsgProcessingSpecTest{
 		Name:          "different proposal round change justification",
 		Pre:           pre,
-		PostRoot:      "eeee6a5783cec81c67ffe6a185f9afb4c775b80a815f22dbb32489f8ffa6adce",
+		PostRoot:      "020830fcec4b0feffbcbf86465926fe80657ce62d1dc637cc141856e4013d6ef",
 		InputMessages: msgs,
-		OutputMessages: []*qbft.SignedMessage{
-			testingutils.TestingPrepareMessageWithRound(ks10.Shares[1], types.OperatorID(1), 3),
+		OutputMessages: []*types.SignedSSVMessage{
+			testingutils.TestingPrepareMessageWithRound(ks.OperatorKeys[1], types.OperatorID(1), 3),
```

Comment:
True!
But reading through the test it seems @MatheusFranco99 was correct with this change...
The reason it passed before is because the tests don't verify sigs, they only compare signing roots